ci/stress_tests/application_cluster_template.yaml

```diff
@@ -37,7 +37,7 @@ provider:
     # Availability zone(s), comma-separated, that nodes may be launched in.
     # Nodes are currently spread between zones by a round-robin approach,
     # however this implementation detail should not be relied upon.
-    availability_zone: us-west-2a,us-west-2b
+    availability_zone: us-west-2b
```

Comment:
The GPU instances are not available in `us-west-2a`, so when it tries to create them there it hangs.